qtop.py

```diff
@@ -1560,7 +1560,11 @@ def print_y_lines_of_file_starting_from_x(self, file, x, y):
         Justification for implementation:
         http://unix.stackexchange.com/questions/47407/cat-line-x-to-line-y-on-a-huge-file
         """
-        return 'clear;tail -n+%s %s | head -n%s' % (x, file, y)
+        temp_f = tempfile.NamedTemporaryFile(delete=False, suffix='.out', dir=TMPDIR)
+        pre_cat_command = '(clear;tail -n+%s %s | head -n%s) > %s' % (x, file, y, temp_f.name)
+        _ = subprocess.call(pre_cat_command, stdout=stdout, stderr=stdout, shell=True)
+        cat_command = 'clear;cat %s' % temp_f.name
```

Comment:
2 times using clear, is that really needed? you may need `cat -v` to get it working, eliminating the above `clear`

N.B. doing that eliminates one more process invocation, therefor it's faster. And therefor desired!